### PR TITLE
add 35 search-insert-position

### PR DESCRIPTION
**Pull request by kobakobashu**
time: 5 m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code.

File: 35.search-insert-position.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# @lc code=start
+class Solution:
+    def searchInsert(self, nums: List[int], target: int) -> int:
+        """
+        time: O(len(nums))
+        space: O(1)
+        """
+        for index in range(len(nums)):
+            #print(index)
```
Comment (ryuji0123) [inline 35.search-insert-position.py:15]: delete this

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I fixed my code.  Please check it.

**Review by ryuji0123 — APPROVED** [review]
LGTM

**Pull request merged**